docs/DEVELOPER_GUIDELINES.md

```diff
@@ -0,0 +1,5 @@
+# Developer Guidelines
+
+* When using entity constants in files within the `/shared` or `/web-api` directory, directly import them at the top of the file. When using them in files within the `/web-client` directory, retrieve them using `applicationContext.getConstants()`. 
+
+* Don't use a computed value in another computed or set state within a computed. 
```

Comment:
@ericsorenson Any chance you could include an example here? The context for this escapes me.

Reply:
Updated!